apps/frontend/src/app/dashboard/card/card.component.html

```diff
@@ -14,7 +14,7 @@ <h5 class="card-title">{{ poll.title }}</h5>
       </div>
     </div>
   </div>
-  <div class="card-body">
+  <div class="card-body p-2">
```

Comment:
Why this change?

Reply:
Cause otherwise something like this might happen on small devices:
![image](https://github.com/Morphclue/apollusia/assets/43741877/4a3a4f7c-27b7-4ea7-b2e8-1c833d47d457)

Reply:
But the smaller padding looks a little off imo